frontend/src/app/modules/home/projects/create-project/create-project.component.ts

```diff
@@ -0,0 +1,180 @@
+import { Component, OnInit } from '@angular/core';
+import { MenuItem } from 'primeng/api';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { ToastNotificationService } from '@core/services/toast-notification.service';
+
+@Component({
+    selector: 'app-create-project',
+    templateUrl: './create-project.component.html',
+    styleUrls: ['./create-project.component.sass']
+})
+export class CreateProjectComponent implements OnInit {
+    // This class is for the test/example, it's not a final result !!!!
+    public formGroup: FormGroup;
+    public tabItems: MenuItem[];
+    public activeTabItem: MenuItem;
+    public viewPlatformCards: {
+        img: string, name: string, isBrowser?: boolean, isServer?: boolean, isMobile?: boolean, isDesktop?: boolean
+    }[];
+    public selectedAlertCategory: any = null;
+    public alertCategories = [
+        { name: 'I\'ll create my own alerts later' },
+        { name: 'Alert me on every new issue' },
+        { name: 'When there are more than' }];
+    public alertsCount: number = 10;
+    public alertTypes = [
+        { name: 'occurrences of' },
+        { name: 'users affected by' }
+    ];
+    public selectedAlertType: any = null;
+    public alertTimeIntervals = [
+        { name: 'one minute' },
+        { name: '5 minutes' },
+        { name: '15 minutes' },
+        { name: 'one hour' },
+        { name: 'one day' },
+        { name: 'one week' },
+        { name: '30 days' }
+    ];
+    public userTeams = [
+        { name: '#vitaliy-shatskiy' },
+        { name: '#binary-studio' },
+        { name: '#watch-dog' },
+        { name: '#ats' },
+    ];
```

Comment:
It would be good to move all those hard-coded values at least to separate file

Reply:
moved into fake-data.ts